setup.py

```diff
@@ -33,7 +33,6 @@ def readme():
         'Operating System :: MacOS',
         'Operating System :: Unix',
         'Programming Language :: Python',
-        'Programming Language :: Python :: 2',
```

Comment:
It's still Python 2 compatible, why would you remove it?

Reply:
There are bad edge cases where someone could make this mistake:
```
circuit(5, 5, "mybreaker") # bad. will run, but fail at an unknown time later.
circuit(5,5, "mybreaker", Exception) # also bad.
circuit(5, 5, name="mybreaker") # ok
circuit(5, 5, Exception, "mybreaker") # ok
```
There is an assertion to guard against this, but it requires special case logic for checking strings in Python2 vs Python3.

I re-added the 2 specifier and the special case.